src/main/java/org/apache/commons/imaging/ImageParser.java

```diff
@@ -682,8 +688,10 @@ public abstract Dimension getImageSize(ByteSource byteSource, Map<String, Object
      *                            parser implementation.
      * @throws IOException        In the event of unsuccessful read or access operation.
      */
-    public abstract String getXmpXml(ByteSource byteSource, Map<String, Object> params)
-            throws ImageReadException, IOException;
+    public String getXmpXml(final ByteSource byteSource, final Map<String, Object> params)
```

Comment:
I'm not sure whether it is a good idea to have this method here if it doesn't make sense for all formats (as stated in the JavaDoc)

Reply:
Reading the issue, and this last comment, I agree about the issue being actually a bad design. IMO, we should move that method to an interface, perhaps ``. Then implementations can choose to implement it when appropriate.

So for `PcxImageParser` it would simply not implement it.